### PR TITLE
build: use GNU build system

### PR DESCRIPTION
**Pull request by giuseppe**
so we can do: ./configure && make && make install for installing the package

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,8 +3,10 @@
 ## Install
 
 ```console
+$ ./autogen.sh
+$ ./configure --prefix=/usr
 $ make
-$ mv slirp4netns ~/bin
+$ make install
```
Comment (AkihiroSuda) [inline README.md]: Maybe `--prefix=$HOME` should be default?

Comment (AkihiroSuda) [reply README.md]: s/default/recommended config/

Comment (giuseppe) [reply README.md]: That will install the program under `$HOME/bin`.  Is that the preferred location?

**Comment by AkihiroSuda** [comment]
@giuseppe could you update `.gitignore`? I'm not familiar with autotools.

**Comment by giuseppe** [comment]
@AkihiroSuda done: https://github.com/AkihiroSuda/slirp4netns/pull/6

Also it fixes a build issue that I've introduced, and it is visible on a clean tree without temporary .o files

**Pull request merged**